signature/policy_eval_signedby_test.go

```diff
@@ -113,14 +117,17 @@ func TestPRSignedByIsSignatureAuthorAccepted(t *testing.T) {
 	pr, err = NewPRSignedByKeyData(ktGPG, []byte{}, prm)
 	require.NoError(t, err)
 	// Pass nil pointers to, kind of, test that the return value does not depend on the parameters.
-	sar, parsedSig, err = pr.isSignatureAuthorAccepted(nil, nil)
-	assertSARRejectedPolicyRequirement(t, sar, parsedSig, err)
+	sar, parsedSig, err = pr.isSignatureAuthorAccepted(m, nil, nil)
+	// FIXME: The error is: errors.InvalidArgumentError("no armored data found") but for
+	// some reason it is surfaced here and not translated to PolicyRequirementError.
+	// However the sar is rejected...
+	assertSARRejectedPolicyRequirement(t, sar, parsedSig, PolicyRequirementError(""))
```

Comment:
> Though arguably the `len(trustedIdentities) == 0` is an invalid `PolicyRequirement`, not a failure to fulfill it… it really isn’t that well defined.

Looking at other cases, that should indeed be a `PolicyRequirementError`. What to do on corrupt keyring data is unclear.